backend/app/main.py

```
@@ -0,0 +1,24 @@
+app.include_router(uploads.router, prefix="/upload", tags=["uploads"])
+
+
+@app.get("/")
+async def root():
+    return {"message": "Welcome to NetPack API"}
+
+
+if __name__ == "__main__":
+    import uvicorn
+
+    uvicorn.run(app, host="0.0.0.0", port=8000)
```

Ruff (0.15.15)

[error] Possible binding to all interfaces

(S104)